kubernetes/polkassembly/values-polkadot-prod.yaml

```diff
@@ -0,0 +1,46 @@
+# PARITY-PROD values for polkassembly.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+chainName: polkadot
+nodeSelector: { cloud.google.com/gke-nodepool: apps-pool }
+
+frontend:
+  ingress:
+    enabled: true
+    annotations:
+      traefik.ingress.kubernetes.io/redirect-entry-point: https
+    hosts:
+      - host: polkadot.polkassembly.io
+        path: /
+      - host: polkassembly.io
+        path: /
+    tls: []
+
+authServer:
+  config:
+    DOMAIN_NAME: "polkadot.polkassembly.io"
+    PROPOSAL_BOT_USER_ID: "1"
+chainDbWatcher:
+  config:
+    PROPOSAL_BOT_USER_ID: "1"
+    CHAIN_DB_GRAPHQL_URL: "http://nodewatcher-server:4000"
+
+healthMonitor:
+  config:
+    # Listening Port
+    PORT: 8034
+    # polkassembly auth-server
+    AUTH_SERVER: "http://auth-server-service:8010"
+    # polkassembly hasura graphql service
+    HASURA_SERVER: "http://hasura-service:8080"
+    # polkassembly hasura chain-db service
+    CHAIN_DB_WATCHER_SERVER: "http://chain-db-watcher-service:8019"
+    # nomidot backend server ( CHAIN_DB_SERVER == CHAIN_DB_GRAPHQL_URL )
+    CHAIN_DB_SERVER: "http://nodewatcher-server:4000"
```

Comment:
I'm not sure about this one and the following one. I touched these 2 (for kusama) in https://github.com/paritytech/polkassembly/pull/924